samples/relocate_vm.py

```diff
@@ -0,0 +1,202 @@
+from __future__ import print_function
```

Comment:
Add License, Author and  Sample description documentation is required. Please refer other samples